eva-accession-deprecate/src/test/java/uk/ac/ebi/eva/accession/deprecate/configuration/DeprecateClusteredVariantsStepConfigurationTest.java

```diff
@@ -55,9 +55,9 @@
 
     private static final String DBSNP_CLUSTERED_VARIANT_ENTITY_DECLUSTERED = "dbsnpClusteredVariantEntityDeclustered";
 
-    private static final long EXPECTED_VARIANTS_TO_BE_FULLY_DECLUSTERED = 2;
+    private static final long EXPECTED_VARIANTS_TO_BE_NOT_FULLY_DECLUSTERED = 4;
```

Comment:
Why is the variable renamed? Those 4 won't be fully declustered?

Reply:
if you see the previous assertion, it doesn't make sense. after deprecating, what will remain in the DCVEDeclustered collection will be those that weren't completely declustered. the number changes because I added more entries to the test data so that there are more scenarios.
```
assertEquals(EXPECTED_VARIANTS_TO_BE_FULLY_DECLUSTERED,
             mongoTemplate.getCollection(DBSNP_CLUSTERED_VARIANT_ENTITY_DECLUSTERED).count());
```